apps/antalmanac/package.json

```
@@ -18,7 +18,8 @@
         "deploy": "gh-pages -d build",
         "format": "prettier --write src",
         "lint": "eslint src",
-        "test": "vitest"
+        "test:watch": "vitest",
+        "test": "vitest run"
```

I'm going to use this command instead because it's the simplest; to invoke `vitest run`, use `pnpm test run`

```suggestion
        "test": "vitest"
```